solr/core/src/java/org/apache/solr/util/ExportTool.java

```diff
@@ -390,6 +400,8 @@ void exportDocs() throws Exception {
             //ignore
           }
         }
+        System.out.println("\nTotal Docs exported: "+ (docsWritten.get() -1)+
```

Comment:
@noblepaul  - the precommit failed due to the use of `System.out`

Reply:
fixing...